handbook/engineering/README.md

```diff
@@ -36,6 +36,15 @@ To [create an engineering-initiated user story](https://fleetdm.com/handbook/eng
 
 > We prefer the term engineering-initiated stories over technical debt because the user story format helps keep us focused on our users and contributors.
 
+### Manage release branches
+Every three weeks, release a minor version of Fleet from the `main` branch. 
+
+Every week between minor releases, release a patch release with fixes for released bugs. 
+
+When the next release is a patch, create a patch branch off the latest tagged release of Fleet. Merge released bug fixes directly into the target patch branch to avoid merge conflicts later in the release cycle. After merging into the patch branch, submit a PR to `main` containing the same fix and resolve any merge conflicts.
```

Comment:
I think we are missing a mention to developers that when they start to work on fixing a `~released_bug` they should branch off the current most up to date patch branch. To minimize confusion we should clear and maintain no other patch-* branches in the repo

Reply:
Then the final caveat is what do we do if we decide it will now be a minor update that includes a high priority feature. Maybe there needs to be a callout that if a feature is merged in it needs to move to a new branch after the merge.

Reply:
Good point. I've added a section below called "Fix a bug" that defines the process based on released vs. unreleased bugs. 

We can't delete the branches (ever) because we are tagging from the branch, so the tag will always point to the commit in the branch. This is a risk area; as managers, we must be very clear about the latest patch branch. To make it easier, we can tell folks it's always the newest version you can find. 

We can protect the previously published patch branches. While that won't stop folks from branching from the wrong branch, it will stop them from merging back into it.

Reply:
I checked, and you're right. We don't need the branch; the tag stays in history. 

In that case, we can update the patch release process to have a step of deleting the patch branch after publishing the release.

Reply:
https://github.com/fleetdm/fleet/branches/stale also there is a plethora of other branches we can most likely clean out (reduce info to fetch) (not as important though)

Reply:
Only 582 to clear out 😆 I think we'll need a script for that.